README.md

```diff
@@ -2,15 +2,15 @@
 
 <p><a  href="https://vote.makerdao.com/"><img  alt="maker logo"  src="./maker-logo.png"  alt="vote.makerdao.com"  width="125"></a></p>
```

Comment:
Should this icon be updated to the Sky one, as well as the gov portal URL here and below in this file?